client/my-sites/checkout/composite-checkout/components/checkout-order-banner/index.tsx

```diff
@@ -0,0 +1,18 @@
+import { useShoppingCart } from '@automattic/shopping-cart';
+import * as React from 'react';
```

Comment:
Minor: you shouldn't need this import. React is imported automatically by our build system.

Reply:
Removed. Thanks! ☺️